src/main/java/org/jabref/logic/importer/util/FileFieldParser.java

```diff
@@ -77,17 +82,25 @@ private static LinkedFile convert(List<String> entry) {
             }
         }
 
-        if (field == null) {
-            field = new LinkedFile(entry.get(0), Path.of(entry.get(1)), entry.get(2));
-        }
+        try {
+            if (field == null) {
+                field = new LinkedFile(entry.get(0), Path.of(entry.get(1)), entry.get(2));
+            }
 
-        // link is only mandatory field
-        if (field.getDescription().isEmpty() && field.getLink().isEmpty() && !field.getFileType().isEmpty()) {
-            field = new LinkedFile("", Path.of(field.getFileType()), "");
-        } else if (!field.getDescription().isEmpty() && field.getLink().isEmpty() && field.getFileType().isEmpty()) {
-            field = new LinkedFile("", Path.of(field.getDescription()), "");
+            // link is only mandatory field
+            if (field.getDescription().isEmpty() && field.getLink().isEmpty() && !field.getFileType().isEmpty()) {
+                field = new LinkedFile("", Path.of(field.getFileType()), "");
+            } else if (!field.getDescription().isEmpty() && field.getLink().isEmpty() && field.getFileType().isEmpty()) {
+                field = new LinkedFile("", Path.of(field.getDescription()), "");
+            }
+            entry.clear();
+        }catch (InvalidPathException exception){
+            DefaultTaskExecutor.runInJavaFXThread(() ->
+                    JabRefGUI.getMainFrame()
```

Comment:
I believe this causes a architecture violation. Can you instead catch the InvalidPathException in the closest gui package caller method?

Reply:
Shouldn't this just return a "unparsed" LinkedFile? @koppor mabye can clarify that point.